pySDC/tests/test_helpers.py

```diff
@@ -81,3 +85,136 @@ def get_derived_from_in_package(base_class, base_package):
                 derived.append(cls)
 
     return derived
+
+
+@pytest.mark.skip(reason="This function is called from a different test with the desired parameters")
```

Comment:
I am a pytest noob, so maybe this is not the desired way of doing this, but pytest wants to call every function in the test file, I think. Since this function is called from the other test, I don't want to test it once more and so I marked it to be skipped. If you don't want any skipping, another option is to move the test of one parameter combination here.